### PR TITLE
Update product name substitution in reference section

### PR DESCRIPTION
**Pull request by krisdale**
-Replace instances of CLOSIA substitution with CL-ATTR.
-Replace in-line use of Clear Linux with CL.
-Clean up white space.
-Limited clean up in collab section, in pref of future rework
-Set landing page toctree to use maxdepth 1

Signed-off-by: Kristal Dale <kristal.dale@intel.com>

### CODE REVIEW

**Comment by krisdale** [comment]
@mvincerx Please review at your convenience. This is a maintenance request, primarily around phasing out the old substitution.

File: source/clear-linux/reference/collaboration/documentation/code.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Code blocks
 ###########
 
-Collaborating to the |CLOSIA| is all about code. Therefore, your
+Collaborating to the |CL-ATTR| is all about code. Therefore, your
```
Comment (mvincerx) [inline source/clear-linux/reference/collaboration/documentation/code.rst]: Please change "Collaborating to" to "Contributing to" in Line 6.

File: source/clear-linux/reference/bundles/bundles.rst [hunk 2]
```diff
@@ -3,8 +3,8 @@
 Available bundles
 #################
 
-This document provides a current list of `available bundles`_ as 
-of ``12 September 2018``.                                      
+This document provides a current list of `available bundles`_ as
```
[the remark this answers is not in the document]
Comment (krisdale) [reply source/clear-linux/reference/bundles/bundles.rst]: As we discussed, I will not remove this file. This PR will be merged prior to PR 309, which will replace my bundles.rst.

File: source/clear-linux/reference/collaboration/documentation/code.rst (hunk 1 above)
Comment (krisdale) [reply source/clear-linux/reference/collaboration/documentation/code.rst]: Done.

**Comment by krisdale** [comment]
@mvincerx Changes made as we discussed.

File: source/clear-linux/reference/bundles/bundles.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (mvincerx) [reply source/clear-linux/reference/bundles/bundles.rst]: Yes. Sounds good. Timing is everything. Thanks.

**Comment by mvincerx** [comment]
LGTM.

**Pull request merged**